### DISCUSSION TITLE
1.3.3

### DISCUSSION THREAD

**Discussion by louislam** [louislam/dockge#272] [Announcements]
### 🐛 Bug Fixes
- #255 Fix: The .env editor is stopped working if the file contains `\r\n` (Thanks @syko9000)
- #269 Fix: Check update is not working 
- #268 Envsubst the yaml file in order to display the correct values in the UI instead of tags such as `${PORT}`
- #270 Fix: .env code change is not reactive 

### 🦎 Translation Contributions
- #257 Translations update from Kuma Weblate (Thanks @thetredev @demonisius @GunnarNorin @ti777777 @altendorfme)

### ⬆️ Security Fixes
- https://github.com/louislam/uptime-kuma/security/advisories/GHSA-88j4-pcx8-q4q3
- https://github.com/louislam/uptime-kuma/security/advisories/GHSA-mj22-23ff-2hrr

### Others
- Other small changes, code refactoring and comment/doc updates in this repo:
#264 (Thanks @derekmahar)

Please let me know if your username is missing, if your pull request has been merged in this version, or your commit has been included in one of the pull requests.

<hr /><em>This discussion was created from the release <a href='https://github.com/louislam/dockge/releases/tag/1.3.3'>1.3.3</a>.</em>

**Comment by derekmahar** [comment]
Thank you for mentioning me for my two tiny documentation corrections!
